src/Raven.Studio/typescript/viewmodels/database/settings/connectionStrings.ts

```diff
@@ -137,8 +137,10 @@ class connectionStrings extends viewModelBase {
         }    
     }   
 
-    isConnectionStringInUse(connectionStringName: string) :boolean {
-        return _.includes(Object.keys(this.connectionStringsTasksInfo), connectionStringName);       
+    isConnectionStringInUse(connectionStringName: string, task: string): boolean {
+        if (this.connectionStringsTasksInfo[connectionStringName].find(x => x.TaskType === task))
```

Comment:
```suggestion
        return this.connectionStringsTasksInfo[connectionStringName].find(x => x.TaskType === task));
```

Reply:
the logic I used here is not right, I have changed this so pls check in next PR.